### ISSUE TITLE
Issue when liking/disliking videos

### ISSUE THREAD

**Issue by syndiate** [Rehike/Rehike#297]
### Rehike version

0.7

### Operating system

Windows 10 Home 1909 build 18363.1556

### PHP version

8.1.6

### What's going on?

When I try to like/dislike videos, the service_ajax endpoint returns this stack trace:

```
<br />
<b>Fatal error</b>:  Uncaught (in promise) Rehike\Exception\Network\InnertubeFailedRequestException in C:\Program Files\xampp\htdocs\modules\Rehike\Network.php:126
Stack trace:
#0 C:\Program Files\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\Promise.php(274): Rehike\Network::Rehike\{closure}(Object(YukisCoffee\CoffeeRequest\Network\Response))
#1 C:\Program Files\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\Util\QueuedPromiseResolver.php(57): YukisCoffee\CoffeeRequest\Promise-&gt;resolve(Object(YukisCoffee\CoffeeRequest\Network\Response))
#2 C:\Program Files\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\Loop.php(240): YukisCoffee\CoffeeRequest\Util\QueuedPromiseResolver-&gt;finish()
#3 C:\Program Files\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\Loop.php(230): YukisCoffee\CoffeeRequest\Loop::finishQueuedPromises()
#4 C:\Program Files\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\Loop.php(84): YukisCoffee\CoffeeRequest\Loop::cleanup()
#5 C:\Program Files\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\CoffeeRequest.php(121): YukisCoffee\CoffeeRequest\Loop::run()
#6 C:\Program Files\xampp\htdocs\modules\Rehike\Network.php(214): YukisCoffee\CoffeeRequest\CoffeeRequest::run()
#7 C:\Program Files\xampp\htdocs\controllers\core\HitchhikerController.php(150): Rehike\Network::run()
#8 C:\Program Files\xampp\htdocs\modules\Rehike\ControllerV2\GetControllerInstance.php(51): Rehike\Controller\core\HitchhikerController-&gt;post(Object(Rehike\YtApp), '', Object(Rehike\ControllerV2\RequestMetadata))
#9 C:\Program Files\xampp\htdocs\modules\Rehike\ControllerV2\GetControllerInstance.php(73): Rehike\ControllerV2\GetControllerInstance-&gt;_cv2WrappedCallControllerMethod('post', Array)
#10 C:\Program Files\xampp\htdocs\modules\Rehike\ControllerV2\Router.php(209): Rehike\ControllerV2\GetControllerInstance-&gt;__call('post', Array)
#11 C:\Program Files\xampp\htdocs\modules\Rehike\ControllerV2\Router.php(156): Rehike\ControllerV2\Router::pointerHandler('ajax/service', 'post')
#12 C:\Program Files\xampp\htdocs\modules\Rehike\ControllerV2\Router.php(38): Rehike\ControllerV2\Router::baseRequestMethod(Array, 'post')
#13 C:\Program Files\xampp\htdocs\router.php(105): Rehike\ControllerV2\Router::post(Array)
#14 C:\Program Files\xampp\htdocs\modules\Rehike\Boot\Bootloader.php(44): require('C:\\Program File...')
#15 C:\Program Files\xampp\htdocs\modules\Rehike\Boot\Bootloader.php(22): Rehike\Boot\Bootloader::postboot()
#16 C:\Program Files\xampp\htdocs\index.php(61): Rehike\Boot\Bootloader::startSession()
#17 {main}
  thrown in <b>C:\Program Files\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\Exception\UncaughtPromiseException.php</b> on line <b>38</b><br />
```

**Comment by aubymori** [comment]
why the fuck is the response blank? the fuck???

**Comment by aubymori** [comment]
[response.zip](https://github.com/Rehike/Rehike/files/11782627/response.zip)
managed to get a `var_dump` of the response object. it is FOURTY-ONE (41) MEGABYTES which is why i had to compress it

**Comment by aubymori** [comment]
yeah turns out i'm fucking stupid they changed the endpoint from `action` to `like/action` so now it's `like/like`, `like/removelike`, etc. etc.

**Comment by aubymori** [comment]
Fixed in 8034c3ab56bfb5fdf57a72626d27f3254c06e331. Closing.
